src/player/lib/audio-manager/HowlerAudioManager.ts

```diff
@@ -64,6 +66,12 @@ export class HowlerAudioManager implements IAudioManager {
       onseek: action(() => {
         this.currentTime = this.getCurrentTime();
       }),
+
+      onvolume: action(() => {
+        const volume = this.howl?.volume();
+
+        this.currentVolume = typeof volume === "number" ? volume : 0;
```

Comment:
The following code would work? I tend to think it is more readable, what do you think?
```
this.currentVolume = volume ?? 0;
```

Reply:
Good point 👍